static/src/js/reducers/__tests__/collectionsResults.test.js

```diff
@@ -58,15 +58,14 @@ describe('FINISHED_COLLECTIONS_TIMER', () => {
       type: FINISHED_COLLECTIONS_TIMER
     }
 
-    // Set current time to 10, and future time to 15
-    // Load time will equal 5
+    // Set start time to 10, current time to 15 and expect a difference of 5
```

Comment:
```suggestion
    // Set start time to 10, current time to 15 and expect a difference of 5
    // Load time will equal 5
```
seems like that comment from before was still accurate but, also is kind of unnecessary I'm good either way

Reply:
Yeah this is in response to Drew's earlier feedback. Originally I copied this comment line from this old test for my new test. I reworded it based on his feedback, then realized that I had gotten it from the older test, so then I edited the old one to match. Either way works for me.